tests/cleanup/Makefile

```diff
@@ -0,0 +1,2 @@
+clean_cloud_envs:
+	find -type f -executable -exec {} \;
```

Comment:
This target needs to be in the master Makefile, Jenkins knows to look only in there.